src/kademlia/get_item.cpp

```diff
@@ -78,7 +88,7 @@ void get_item::got_data(bdecode_node const& v,
 		// and it's immutable
 
 		m_data.assign(v);
-		bool put_requested = m_data_callback(m_data);
+		bool put_requested = m_data_callback(m_data, false);
```

Comment:
shouldn't this be true? Or, would this ever happen? We only pass the data on if it matches the hash, and this is empty.. it will never be anything but empty.. I'm a bit confused by this path to be honest..